webhooks/virtualmachine/mutation/scsi_controller.go

```diff
@@ -0,0 +1,252 @@
+// © Broadcom. All Rights Reserved.
```

Comment:
Rename this file to `virtualmachine_mutator_hardware_controllers.go`. Do the same for the associated test file. Following patterns in a project is important.